pkg/cmd/experimental/syncgroups/syncgroups.go

```diff
@@ -88,25 +75,12 @@ type SyncGroupsOptions struct {
 	// Source determines the source of the list of groups to sync
 	Source GroupSyncSource
 
-	// Scope determines the scope of the group sync
-	Scope GroupSyncScope
-
-	// ConfigSource is the path to the sync config
-	ConfigSource string
-
 	// Config is the LDAP sync config read from file
 	Config api.LDAPSyncConfig
 
-	// WhitelistSource is the path to the whitelist file, if provided
-	WhitelistSource string
-
 	// WhitelistContents are the contents of the whitelist: names of OpenShift group or LDAP group UIDs
 	WhitelistContents []string
 
-	// SyncExisting determines that only groups in OpenShift previously synced with this LDAP server
-	// will be synced again
-	SyncExisting bool
-
 	// Confirm determines whether not to write to openshift
 	Confirm bool
```

Comment:
> Dry-run?

I debated, but decided to match the flag for ease of binding.  We've previously decided that commands that default to `--dry-run=true`, should have a `--confirm` flag instead (it allows `oc --confirm` instead of `oc --dry-run=false`.).  `Confirm` looked stupid in the syncer, but I'm open to other names in the syncer if you want to avoid the later `!`.